game.cpp

```diff
@@ -440,6 +448,11 @@ player game::wall_collision(player p)
   return p;
 }
 
+std::default_random_engine& game::get_rng() noexcept
```

Comment:
I have a limited understanding of this but do we really want to pass the RNG by reference? Isn't there a risk it is overwritten through the game?

Reply:
Gotcha!